lib/components/home/artist_rails/artist_card.js

```diff
@@ -8,19 +8,26 @@ const { ARTemporaryAPIModule } = NativeModules
 
 import colors from '../../../../data/colors'
 import ImageView from '../../opaque_image_view'
-import SwitchBoard from '../../../modules/switch_board'
+import SwitchBoard from '../../../native_modules/switch_board'
 import InvertedButton from '../../buttons/inverted_button'
-import Events from '../../../modules/events'
+import Events from '../../../native_modules/events'
+
+// $FlowFixMe: Flow doesn't know RN provides this
+import { Promise } from 'react-native'
 
 class ArtistCard extends React.Component {
   state: {
     processingChange: boolean,
+    following: ?boolean,
+    followStatusChanged: ?boolean
   }
 
   constructor(props) {
     super(props)
     this.state = {
       processingChange: false,
+      followStatusChanged: null,
+      following: null
     }
```

Comment:
could also have made `following: ?boolean` be `following?: boolean` which would mean that `following`  could optionally be inside state, but I think an optional for following sets the right mental context